classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  attr_accessor :label
```

Comment:
- [OPTIONAL] in order to follow the best practices please make sure to move the accessor up before the initialize method